README.md

```diff
@@ -1,37 +1,113 @@
-[![Check unchanged](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/check_unchanged.yml/badge.svg)](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/check_unchanged.yml)
```

Comment:
Why do we remove this?

Reply:
because 
![image](https://github.com/welfare-state-analytics/riksdagen-corpus/assets/8887511/3afacd95-64e0-4bee-96c0-20cccf088649)

Reply:
this workflow run was incorporated into the unit tests. The badge doesn't link to anything -- here's the readme from main as it shows up now.
![image](https://github.com/welfare-state-analytics/riksdagen-corpus/assets/8887511/c3c5358b-a59d-42eb-b515-af62f7a7b215)

Reply:
Ah ok. So its a bug in the readme?

Reply:
exactly